server/hangouts/publications.js

```diff
@@ -82,3 +82,22 @@ Meteor.publish( 'hangoutSearch', function(searchTerm) {
 
     return Hangouts.search(searchTerm);
 });
+
+Meteor.publish('hangoutParticipants', function(room) {
```

Comment:
`hangoutParticipants` and `allHangoutParticipants` These publications should be also placed under `/server/app_stats/publications.js` directory.

Reply:
Fixed